packages/ai-ide/src/browser/coder-agent.ts

```diff
@@ -38,5 +41,25 @@ export class CoderAgent extends AbstractStreamParsingChatAgent {
     override promptTemplates = [getCoderReplacePromptTemplate(true), getCoderReplacePromptTemplate(false)];
     override functions = [GET_WORKSPACE_DIRECTORY_STRUCTURE_FUNCTION_ID, GET_WORKSPACE_FILE_LIST_FUNCTION_ID, FILE_CONTENT_FUNCTION_ID, WriteChangeToFileProvider.ID];
     protected override systemPromptId: string | undefined = CODER_REPLACE_PROMPT_TEMPLATE_ID;
-
+    override async invoke(request: MutableChatRequestModel): Promise<void> {
+        await super.invoke(request);
+        this.suggest(request);
+    }
+    async suggest(context: ChatSession | ChatRequestModel): Promise<void> {
+        const model = ChatRequestModel.is(context) ? context.session : context.model;
+        const session = this.chatService.getSessions().find(candidate => candidate.model.id === model.id);
+        if (!(model instanceof MutableChatModel) || !session) { return; }
+        if (model.isEmpty()) {
+            model.setSuggestions([
+                {
+                    kind: 'callback',
+                    callback: () => this.chatService.sendRequest(session.id, { text: '@Coder please look at #_f and fix any problems.' }),
+                    content: '[Fix problems](_callback) in the current file.'
+                },
+            ]);
+        } else {
+            model.setSuggestions([new MarkdownStringImpl(`Keep chats short and focused. [Start a new chat](command:${AI_CHAT_NEW_CHAT_WINDOW_COMMAND.id}) for a new task`
```

Comment:
@planger Should suggestions be part of the session or the model?

Reply:
Placed there for now because it's currently where change sets and context also live.